sphinxcontrib/openapi/openapi20.py

```diff
@@ -46,10 +46,18 @@ def _httpresource(endpoint, method, properties):
         for line in param.get('description', '').splitlines():
             yield '{indent}{indent}{line}'.format(**locals())
 
+    # print the json body params
+    for param in filter(lambda p: p['in'] == 'body', parameters):
+        if 'schema' in param:
+            yield ''
+            for line in convert_json_schema(param['schema']):
+                yield '{indent}{line}'.format(**locals())
+            yield ''
+
     # print response status codes
-    for status, response in responses.items():
+    for status, response in sorted(responses.items()):
         yield '{indent}:status {status}:'.format(**locals())
-        for line in response['description'].splitlines():
+        for line in response.get('description', '').splitlines():
```

Comment:
Why have you changed `__getitem__` to `.get` here? According to OpenAPI v2 spec ([link]), `description` is required attribute of response object. 

[link]: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#response-object